### ISSUE TITLE
different and new AAK-lists

### ISSUE THREAD

**Issue by DJCrashdummy** [collinbarrett/FilterLists#117]
historical collected:... (the whole story: https://github.com/jspenguin2017/uBlockProtector/issues/237#issue-231505624)
- the original aak-list (_[AakList (Anti-Adblock Killer)](https://github.com/reek/anti-adblock-killer/)_) saw no update since 10 months now...
- as also mentioned in the filterlists (with wrong name for now #115) _[AdBlockProtector | uBlock Protector List](https://github.com/jspenguin2017/uBlockProtector)_ is somehow based on it... (https://github.com/jspenguin2017/uBlockProtector/issues/44)

- and now some months ago a new project [Anti-Adblock Killer Continued](https://xuhaiyang1234.gitlab.io/AAK-Cont/) (AAK-Cont) was started apparently from the same maintainer as _uBlock Protector_ is:
and they also provide 2 filter-lists: https://xuhaiyang1234.gitlab.io/AAK-Cont/#filterlist

since the 3 (or to be precise 4 - no idea why 2 new different ones) lists are somehow closely related but not exact the same, also the the new ones should be listed.

**Comment by Atavic** [comment]
I know that the original aak-list won't be updated. Then there's the _AAK-Cont_ that's hailed as the new one, while jspenguin2017 has his own list that fits nicely with his own _uBlock Protector_ for Chrome and Chromium.

**Comment by Atavic** [comment]
I know that the original aak-list won't be updated. Then there's the AAK-Cont that's hailed as
the new one, while jspenguin2017 has his own list that fits nicely with his own uBlock Protector for Chrome and Chromium.

**Comment by DJCrashdummy** [comment]
maybe ditching _uBlock Protector List_ in favor of having both _AAK-Cont_ lists included (linked against each other with a clear note), since _uBlock Protector List_ is designed for a special addon, makes sense...?

**Comment by Atavic** [comment]
Yes, as AAK-Cont has different versions for various webetensions.

**Comment by Atavic** [comment]
Yes, as AAK-Cont has different versions for various webextensions.

**Comment by xqk94524** [comment]
is AAK-Cont better than AAK?

I'm a bit confused.

**Comment by Atavic** [comment]
AAK is no more. I do not know if AAK Cont on gitlab is active as [uBlockProtector](https://github.com/jspenguin2017/uBlockProtector), but the developer is the same one.

**Comment by collinbarrett** [comment]
Unfortunately, I am having some issues updating the current version of the site. I will try to resolve them so I can perform this update, but I may not get it updated until the new v3 version of the site is launched. Thanks for your patience.

**Comment by collinbarrett** [comment]
If anyone is willing and able to help, I'll accept a PR against [Seed](https://github.com/collinbarrett/FilterLists/tree/master/src/FilterLists.Api/Seed). If not, I'll get to it eventually. Thanks!

**Comment by collinbarrett** [comment]
Just tried to read through the history of @jspenguin2017 's projects, and I'm super confused. Can anyone provide insight as to which unique lists he/she maintains?

Currently, I have the original Reek AAK (see id 207 [here](https://github.com/collinbarrett/FilterLists/blob/master/data/FilterList.json)), which I will keep for posterity.

I also have uBlock Protector (see id 83 [here](https://github.com/collinbarrett/FilterLists/blob/master/data/FilterList.json)), but that looks like it is no more in favor of the nano projects. It looks to me like uBlock Protector should be removed. Or, does @jspenguin2017 still support a fork of the Reek AAK project somewhere?

[These lists](https://github.com/NanoAdblocker/NanoFilters/tree/master/NanoFilters) should be added.

Additionally, a new [syntax](https://github.com/collinbarrett/FilterLists/blob/master/data/Syntax.json) should be added for Nano per [this](https://github.com/NanoAdblocker/NanoCore/blob/master/Notes/FilterIncompatibility.MD) as well as a new [software](https://github.com/collinbarrett/FilterLists/blob/master/data/Software.json) for Nano.

Thanks for any feedback.

**Comment by jspenguin2017** [comment]
I maintain Nano Filters and Nano Defender Integration, none will work properly if installed directly. "uBlock Protector List" should never been listed in the database, nor should "AAK" or "AAK-Cont" as they also depend on their Userscript. 

Nano Filters are designed for Nano Adblocker but they are already hard coded into `assets.json`, there is little to no point in listing them here.

**Comment by collinbarrett** [comment]
Thanks for clarifying. Yeah, I'm still learning this open-source community thing. Thanks for all your work. Many don't have a good grasp on what free and open-source really means. It doesn't mean the developer(s) will do whatever you want them to for you for free. :)

As for the issue in question, I am going to maintain Reek as it was such a mainstay for awhile. With the caveat that it will be made clear that it is discontinued as well as requires the userscript.

I'll drop uBlock Protector.

I'd like to go ahead and index your Nano Filters as well (will be in a separate commit). I plan to show which software and syntaxes each list support, so I will just ensure those are accurate. FilterLists should be a software/syntax-agnostic discovery tool, so happy to have Nano rolled into the fold. It may help drive a bit of interest to your project.

**Comment by arufam124** [comment]
can you create a fersi that does not have a white list because maybe it's a bit of a problem because the white list maybe also has a lot of bad issues and i can not block the wintelist

**Comment by arufam124** [comment]
can you create a version that does not have a white list because maybe it's a bit of a problem because the white list maybe also has a lot of bad issues and i can not block the wintelist

**Comment by collinbarrett** [comment]
@arufam124 FilterLists does not create or maintain any lists. It is just an index of lists that exist. You will need to contact the maintainer of the list you have a question about or create your own fork if you want a specific list variant.

**Comment by Atavic** [comment]
@arufam124

   # Whitelist for Nano Adblocker
   # Nothing for now

NanoWhitelist.txt is void. By the way [NanoCore](https://github.com/NanoAdblocker/NanoCore) is a fork of gorhill's uBo and the entries can be modified by the user.

**Comment by Atavic** [comment]
@arufam124

```
# Whitelist for Nano Adblocker
# Nothing for now
```

NanoWhitelist.txt is void. By the way [NanoCore](https://github.com/NanoAdblocker/NanoCore) is a fork of gorhill's uBo and the entries can be modified by the user.

**Comment by jspenguin2017** [comment]
I added a special settings that discards all non-user whitelists: https://github.com/NanoAdblocker/NanoCore/blob/master/notes/advanced-settings.MD#_nanoignorethirdpartywhitelist-toggleable-experimental
Not sure if this will be useful for @arufam124

**Comment by collinbarrett** [comment]
addressed in #304 . needs evaluation if any changes/additions are still required prior to closing.

**Comment by jspenguin2017** [comment]
The syntax extensions are not currently used in Nano filters (for compatibility), however, the script snippet extensions are. 

Also, the syntax doc is here: https://github.com/NanoAdblocker/NanoDoc/blob/master/en/FilterSyntax.MD

For home page and issues tracker, I prefer https://github.com/NanoAdblocker/NanoFilters and https://github.com/NanoAdblocker/NanoFilters/issues

**Comment by tsugami1** [comment]
is very2 good:
I see here that too many have a white list and block the google I hope this is not harmful to privacy
but I see that many lists of videos are whitelisted
 this resolves my problem on a number of bad adblocker that block videos
